src/app/components/ConsensusTransactionMethod/index.tsx

```diff
@@ -36,31 +38,42 @@ const colorMap = {
     primary: COLORS.grayMedium,
     secondary: COLORS.grayMediumLight,
   },
+  orange: {
+    primary: COLORS.warningColor,
+    secondary: COLORS.warningLight,
+  },
 }
 
-const MethodIcon: FC<MethodIconProps> = ({ icon, label, color = 'blue' }) => {
+const iconRatio = 0.75
+
+export const MethodIcon: FC<MethodIconProps> = ({
+  border = true,
+  color = 'blue',
+  icon,
+  label,
+  size = 40,
+}) => {
   const theme = colorMap[color]
   if (!theme) {
     throw new Error(`Invalid color: ${color}`)
   }
-
   return (
-    <Box gap={3} sx={{ display: 'flex', alignItems: 'center' }}>
+    <Box gap={label ? 3 : 0} sx={{ display: 'inline-flex', alignItems: 'center' }}>
```

Comment:
`gap={label ? 3 : 0}` a more common approach is to make elements conditional. `Gap` already only adds gaps between elements that exist